modules/check_in/app/services/v2/chip/service.rb

```diff
@@ -90,6 +98,19 @@ def identifier_params
         }
       end
 
+      def demographic_confirmations
+        demographics_needs_update = check_in_body[:demographics_up_to_date] ? false : true
+        next_of_kin_needs_update = check_in_body[:next_of_kin_up_to_date] ? false : true
+        {
+          demographicConfirmations: {
+            demographicsNeedsUpdate: demographics_needs_update,
+            demographicsConfirmedAt: Time.zone.now.iso8601,
+            nextOfKinNeedsUpdate: next_of_kin_needs_update,
+            nextOfConfirmedAt: Time.zone.now.iso8601
```

Comment:
nit: Wondering if the payload is `nextOfConfirmedAt` instead of `nextOfKinConfirmedAt`

Reply:
As of now `nextOfConfirmedAt` is correct: https://github.com/department-of-veterans-affairs/clinician_workflow/blob/b8dde1a4d6d89d44f8b03eceda12c448dabc3d6f/openapi/v1/openapi.yaml#L1202

If you want you can try to convince @charleystran to change it :-D

Reply:
:-D thanks for chiming in.  May be this field is common for all nextOf__ :-D.  I will resolve this.